plugins/module_utils/network/sonic/config/vrrp/vrrp.py

```diff
@@ -0,0 +1,719 @@
+
+        if use_v2_checksum is not None and cfg_use_v2_checksum is not None:
+            requests.extend(update_requests(use_v2_checksum, cfg_use_v2_checksum, self.vrrp_config_path['use_v2_checksum'].format(vrid=virtual_router_id)))
+
+        if track_interfaces and cfg_track_interfaces:
+            for track in track_interfaces:
+                interface = track['interface']
+                interface = interface.replace('/', '%2f')
+                for cfg_track in cfg_track_interfaces:
+                    cfg_interface = cfg_track['interface']
+                    cfg_interface = cfg_interface.replace('/', '%2f')
+                    track_url = self.vrrp_config_path['track_interface'].format(vrid=virtual_router_id) + '/vrrp-track-interface=' + interface
+                    requests.extend(update_requests(interface, cfg_interface, track_url))
+        return requests
+
+    def sort_lists_in_config(self, config):
+        if config:
+            config.sort(key=lambda x: x['name'])
+            for cfg in config:
+                if cfg.get('group'):
+                    cfg['group'].sort(key=lambda x: (x['virtual_router_id'], x['afi']))
+                    for group in cfg['group']:
+                        if group.get('virtual_address'):
+                            group['virtual_address'].sort(key=lambda x: x['address'])
+                        if group.get('track_interface'):
+                            group['track_interface'].sort(key=lambda x: x['interface'])
+
+    @staticmethod
+    def get_vip_addresses(vip_addresses_list):
+        """ Get a set of virtual IP/IPv6 addresses available in the given
+        vip_addresses list
+        """
+        vip_addresses = []
+        if not vip_addresses_list:
+            return vip_addresses
+
+        for addr in vip_addresses_list:
+            if addr.get('address'):
+                vip_addresses.append(addr['address'])
+
+        return vip_addresses
+
+    @staticmethod
+    def get_track_interfaces(track_interfaces_list):
+        """ Get a set of track interface groups available in the given
+        track_interfaces list
+        """
+        track_interfaces = []
+        if not track_interfaces_list:
+            return track_interfaces
+
+        for track_interface in track_interfaces_list:
+            if track_interface['interface'] and track_interface['priority_increment']:
+                track_interfaces.append(track_interface)
+
+        return track_interfaces
+
+    @staticmethod
+    def _get_generated_config(commands, have, state):
+        """Get generated config"""
+
+        new_config = remove_empties_from_list(get_new_config(commands, have, TEST_KEYS_formatted_diff))
+        group_default_entries = {
+            'priority': 100,
+            'preempt': True,
+            'advertisement_interval': 1,
+            'version': 2,
+            'use_v2_checksum': False
+        }
```

Comment:
'default_entries' defined in the module can be reused.

Reply:
Instead of traversing through the default_entries and checking for key matching and accessing its default value, 
accessing default values directly through dict looks easier.

Reply:
As discussed, will use "default_attributes"